libs/pageserver_api/src/models.rs

```diff
@@ -356,7 +356,7 @@ pub enum TenantAttachmentStatus {
 
 #[derive(Serialize, Deserialize, Clone)]
 pub struct TenantInfo {
-    pub id: TenantId,
+    pub id: TenantShardId,
```

Comment:
This likely needs a swagger change.

Reply:
Or ... we just call this the `format: hex` :) Unsure if the go generated clients do anything with that information. Perhaps that hex should had never been there and these should just be opaque.

Checked that it does not matter to us because parsing a legacy which is as expected. e2e tests probably will not find anything because the client might be manually regenerated (from a manually refreshed openapi.yaml) so it might be something will break on the next re-generation.

Reply:
Yeah, I think we can take the approach that `hex` was never a standard and means whatever we want it to mean: but I'll open a quick thread with the control plane team to check.